code/modules/client/preference_setup/loadout/items/xeno/machine.dm

```diff
@@ -138,6 +138,8 @@
 	goldendeep["golden deep vest"] = /obj/item/clothing/under/goldendeep/vest
 	goldendeep["golden deep suit"] = /obj/item/clothing/under/goldendeep/suit
 	goldendeep["golden deep skirtsuit"] = /obj/item/clothing/under/goldendeep/skirtsuit
+	goldendeep["golden deep hoplan uniform"] = /obj/item/clothing/under/goldendeep/hoplan
+	goldendeep["golden deep porter uniform"] = /obj/item/clothing/under/goldendeep/porter
```

Comment:
Also generally not a fan of more and more military gear or uniforms being added to the loadout.

Reply:
The Golden Deep prides itself on Vanity. The term "Golden" in the nation's entire name is a testament to this. The citizens within the Golden Deep, in particular the merchants, take what they want to wear and make it part of them. Some people enjoy dressing up in more military-esq apparel in normal life, this is just that but exaggerated because vanity is part of the identity of this nation.

Reply:
I think the design is fine. Golden deep is stylish and flashy - that's understood and accepted.

The main point of contention here that (I hopefully am explaining correctly on Dreamy's behalf) is that the hoplan outfit is a ***military*** uniform.  

The issue is that *we aren't Bay* - We're a civilian ship, so any combat-related outfits should be limited to **security** and **non-ship staff** - regular crewmembers, civillians, should not be able to wear them.

Reply:
The idea we're going off of in this PR is that how, in the similar vein of real life, you can go to a military surplus store and buy real military uniforms for civilian use, that the Golden Deep more than anyone in the galaxy would do something that vain.

Reply:
Tentatively, I'll be limiting the Hoplan uniforms to Consulars and Aides. Changes implemented, also limited them to the Golden Deep culture for good measure, @DreamySkrell.

Reply:
Though for now we'll limit it just to Consular/Diplomatic Aide until I can push out the military lore expansion that goes further into depth about how the Golden Deep uses these kind of things to make money.

Reply:
> you can go to a military surplus store and buy real military uniforms for civilian use

Horizon crew aren't civilians doing civilian things in their private time. They are working on Horizon and representing SCC. They aren't working for, or representing, Golden Deep (other than consulars).

IRL, you can buy military gear, sure, surplus or even active uniforms of any active military. But unless you're actually employed in the army/navy/whatever, you're probably not going to wear that stuff to work.

> Tentatively, I'll be limiting the Hoplan uniforms to Consulars and Aides. Changes implemented, also limited them to the Golden Deep culture for good measure,

That's fine, thank you